setup.py

```diff
@@ -63,6 +86,7 @@ def package_data(pkg, root_list):
         ]
     },
     package_data=package_data("recommender", ["static", "translations"]),
+    install_requires=load_requirements('requirements/base.in'),
```

Comment:
It seems that this package didn't have dependencies before, so  I guess that we don't changes on this file